source/js/fellowships.js

```diff
@@ -136,7 +138,7 @@ function renderFellowsOnDirectoryByTypePage() {
   getFellows({}, fellows => {
     let fellowsByYear = groupFellowsByAttr(`program_year`, fellows);
 
-    let sections = Object.keys(fellowsByYear).sort().reverse().map(year => {
+    let sections = Object.keys(fellowsByYear).filter((year) => parseInt(year, 10) !== CURRENT_YEAR).sort().reverse().map(year => {
```

Comment:
Can you make this `< CURRENT_YEAR`?

Reply:
That is effectively the same thing though?